core/src/test/java/io/wcm/testing/mock/aem/MockLanguageManagerTest.java

```diff
@@ -280,4 +362,10 @@ public void getAdjacentInfo() {
     Map<Locale, LanguageManager.Info> rootPath = languageManager.getAdjacentInfo(context.resourceResolver(), "/");
     assertNull(rootPath);
   }
+
+  public void testGetLanguageRootResource() {
```

Comment:
two empty methods? obsolete or do you wanted to add additional tests?

Reply:
nop auto-generated. Are incorporated in the other tests